docs/END_USER_DOCUMENT.md

```diff
@@ -750,3 +751,38 @@ AsyncBooksAppClient asyncClient = MoccaClient.Builder
         .client(executorClient)
         .build(AsyncBooksAppClient.class);
 ```
+## 8. Request Validation
```

Comment:
Please let's mention somewhere that an `feign.codec.EncodeException` is thrown in case a validation violation happens.

By the way, I have added at #40 that we should remove all mentions of `feign.codec.EncodeException` from end user document, so we remember to come back here and remove it then.

Reply:
do you mean add a note in our documentation or in the comments? I was hesitant to add this to our docs in case we decided to change it.